src/libpmemobj/lane.h

```diff
@@ -115,7 +115,8 @@ struct section_operations {
 	section_destr destroy_rt;
 	section_layout_op check;
 	section_layout_op recover;
-	section_global_op boot;
+	section_global_op init;
```

Comment:
I prefered the `boot` name. We use `init` for persistent structures initialization, for example `heap_boot` vs `heap_init`. You can then either use `destroy` or `cleanup`.